### ISSUE TITLE
FYI: Update packages to avoid security problems!

### ISSUE THREAD

**Issue by nbarbettini** [nbarbettini/SimpleTokenProvider#18]
Microsoft published [this security advisory](https://technet.microsoft.com/library/security/3214296) earlier this month, which describes a security vulnerability in [Microsoft.IdentityModel.Tokens](https://www.nuget.org/packages/Microsoft.IdentityModel.Tokens/) package, which is also used by the [System.IdentityModel.Tokens.Jwt](https://www.nuget.org/packages/System.IdentityModel.Tokens.Jwt/) package.

**Tl;dr - if your code uses either of the above packages, make sure you update to the latest version via NuGet in order to get the security fix.**

The example code in this repository has been updated to point to the latest versions. 👍
